config/app.default.php

```diff
@@ -395,7 +395,7 @@
      * - `autoload` - boolean - (default: false) Whether or not you want an autoloader registered
      */
     'Plugins' => [
-        'BEdita/DevTools' => ['debugOnly' => true, 'bootstrap' => true],
+      'BEdita/DevTools' => ['debugOnly' => true, 'bootstrap' => true],
```

Comment:
Wrong indentation. 🙃  It used to be correct before.

Reply:
Yes, actual reason is: ti was commented and decommented before commit.

But I think it should be commented as default: if composer `require-dev` section is not loaded, this ends up in actual `config/app.php` but no `BEdita/DevTools` plugin is present that case